tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -5277,22 +5277,42 @@ def get_matmul(arr):
 
     bl = randint(0, 2, size=size, dtype= bool)
 
-    integer8 = randint(min_int8, max_int8, size=size, dtype=np.int8)
-    integer16 = randint(min_int16, max_int16, size=size, dtype=np.int16)
-    integer = randint(min_int, max_int, size=size, dtype=int)
-    integer32 = randint(min_int32, max_int32, size=size, dtype=np.int32)
-    integer64 = randint(min_int64, max_int64, size=size, dtype=np.int64)
-
-    fl = uniform(-((abs(min_float) / size[0])**(1/2)), (abs(max_float) / size[0])**(1/2), size = size)
-    fl32 = uniform(-((abs(min_float32) / size[0])**(1/2)), (abs(max_float32) / size[0])**(1/2), size = size)
+    def calculate_max_values(min_for_type, max_for_type):
+        cast = type(min_for_type)
+        min_test = -np.sqrt(abs(min_for_type) / size[0])
+        max_test = np.sqrt(abs(max_for_type) / size[0])
+        print(min_test, max_test, cast(min_test), cast(max_test))
+        return cast(min_test), cast(max_test)
+
+    integer8 = randint(*calculate_max_values(min_int8, max_int8), size=size, dtype=np.int8)
+    integer16 = randint(*calculate_max_values(min_int16, max_int16), size=size, dtype=np.int16)
+    integer = randint(*calculate_max_values(min_int, max_int), size=size, dtype=int)
+    integer32 = randint(*calculate_max_values(min_int32, max_int32), size=size, dtype=np.int32)
+    integer64 = randint(*calculate_max_values(min_int64, max_int64), size=size, dtype=np.int64)
```

Comment:
Can you please explain these changes in the PR description?

Reply:
Oops. I didn't realise I'd committed that file ^^'

Reply:
Done